packages/desktop-client/src/components/budget/rollover/BalanceMenu.tsx

```diff
@@ -42,31 +46,31 @@ export function BalanceMenu({
             onCover?.();
             break;
           default:
-            throw new Error(`Unrecognized menu option: ${name}`);
+            throw new Error(t('Unrecognized menu option: {{name}}', { name }));
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏there's no need to translate these exception messages as they are not shown to the user. Only devs care about these.